### PR TITLE
Stack name fix

### PR DESCRIPTION
**Pull request by JordonPhillips**
This builds off of #3228, adding an alias from the old name to the new name so that anybody attempting to use the old name isn't broken.

cc @kyleknap @jamesls

### CODE REVIEW

File: tests/unit/cloudformation/test_connection.py [hunk 1]
```diff
@@ -444,7 +444,8 @@ def test_describe_stacks(self):
         self.assertEqual(stack.stack_id, 'arn:aws:cfn:us-east-1:1:stack')
         self.assertEqual(stack.stack_status, 'CREATE_COMPLETE')
         self.assertEqual(stack.stack_name, 'MyStack')
-        self.assertEqual(stack.stack_name_reason, None)
+        self.assertEqual(stack.stack_name_reason, '')
+        self.assertEqual(stack.stack_status_reason, '')
```
Comment (jamesls) [inline tests/unit/cloudformation/test_connection.py]: Can we update the response body in this test to have a StackStatusReason so we can verify we're parsing the expected value?

**Comment by jamesls** [comment]
Looks good, had a small comment on the test.

**Comment by JordonPhillips** [comment]
Updated, the value is now `REASON`

**Comment by jamesls** [comment]
:shipit:

**Pull request merged**